### ISSUE TITLE
leveldb requires manual inclusion of shlwapi.lib

### ISSUE THREAD

**Issue by rkcarroll** [microsoft/vcpkg#6874]
Using leveldb (x64-windows), I was not able to get the program to link.  I had to manually add the shlwapi.lib library.

**Comment by wangli28** [comment]
Because the ***config.cmake** file is missing in source and `shlwapi.lib` is a windows library, so you need to manually link `shlwapi.lib`.

**Comment by LilyWangL** [comment]
Hi @rkcarroll, this issue has been fixed in PR #7245. You can use `find_package` to link it. I am closing this issue for now.
```
find_package(leveldb CONFIG REQUIRED)
target_link_libraries(main PRIVATE leveldb::leveldb)
```
